include/ddc/kernels/splines/spline_evaluator.hpp

```diff
@@ -168,6 +279,16 @@ class SplineEvaluator
                 });
     }
 
+    /**
+     * @brief Derivate 1D spline function (described by its spline coefficients) at a given coordinate.
+     *
+     * The spline coefficients represent a 1D spline function in a B-splines (basis splines). They can be obtained using a SplineBuilder. 
+     *
+     * @param coord_eval The coordinate where the spline is derivated. Note that only the component along the dimension of interest is used.
+     * @param spline_coef A Chunkspan storing the 1D spline coefficients.
+     *
+     * @return The derivative of the spline function at the desired coordinate. 
```

Comment:
```suggestion
     * @brief Differentiate a 1D spline function (described by its spline coefficients) at a given coordinate.
     *
     * The spline coefficients represent a 1D spline function in a B-splines (basis splines). They can be obtained using a SplineBuilder.
     *
     * @param coord_eval The coordinate where the spline is differentiated. Note that only the component along the dimension of interest is used.
     * @param spline_coef A Chunkspan storing the 1D spline coefficients.
     *
     * @return The derivative of the spline function at the desired coordinate. 
```

> They can be obtained using a SplineBuilder.

This sentence is repeated multiple times but a `SplineBuilder` is not the only way to obtain spline coefficients. I think that either this comment should be removed or it should be clear that this is not the only option.